docker/Dockerfile

```diff
@@ -1,3 +1,10 @@
+FROM alpine:3.15 AS dl
```

Comment:
Maybe `apkeep` or `apkeep-download` instead of `dl` is a more understandable name?

Reply:
Makes sense, I'll change that!